crates/common/consensus/src/execution_engine/mock_engine.rs

```diff
@@ -25,4 +30,15 @@ impl ExecutionApi for MockExecutionEngine {
     ) -> anyhow::Result<bool> {
         Ok(self.execution_valid)
     }
+
+    async fn engine_get_blobs_v1(
+        &self,
+        blob_version_hashes: Vec<B256>,
+    ) -> anyhow::Result<Vec<Option<BlobsAndProofV1>>> {
+        let mut result = Vec::with_capacity(blob_version_hashes.len());
+        for _i in 0..blob_version_hashes.len() {
+            result.push(None);
+        }
+        Ok(result)
+    }
```

Comment:
```suggestion
    async fn engine_get_blobs_v1(
        &self,
        blob_version_hashes: Vec<B256>,
    ) -> anyhow::Result<Vec<Option<BlobsAndProofV1>>> {
        Ok(blob_version_hashes.into_iter().map(|_| None).collect())
    }
```

we can simplify this